test/evaluation.spec.js

```diff
@@ -158,9 +158,8 @@ module.exports.addTests = function({testRunner, expect}) {
     it('should properly serialize null fields', async({page}) => {
       expect(await page.evaluate(() => ({a: undefined}))).toEqual({});
     });
-    it_fails_ffox('should return undefined for non-serializable objects', async({page, server}) => {
+    it('should return undefined for non-serializable objects', async({page, server}) => {
       expect(await page.evaluate(() => window)).toBe(undefined);
-      expect(await page.evaluate(() => [Symbol('foo4')])).toBe(undefined);
```

Comment:
Why didn’t this one work?

Reply:
This is split into two tests: the Symbol one is now on line 68